src/main/kotlin/com/milaboratory/mixcr/cli/CommandBuildLibrary.kt

```diff
@@ -0,0 +1,394 @@
+            names = ["--d-genes-from-species"],
+            description = ["Species to take Diversity genes from it (human, mmu, lamaGlama, alpaca, rat, spalax"],
+            paramLabel = "<species>",
+            required = true
+        )
+        override var species: String? = null
+            set(value) {
+                ValidationException.requireKnownSpecies(value)
+                field = value
+            }
+    }
+
+    @ArgGroup(exclusive = true, multiplicity = "0..1")
+    var cGenes: CGenesInput? = null
+
+    class CGenesInput : GenesInput {
+        @ArgGroup(exclusive = false)
+        override var fasta: CGenesFasta? = null
+
+        class CGenesFasta : GenesFasta {
+            @set:Option(
+                names = ["--c-genes-from-fasta"],
+                description = ["FASTA file with Constant genes"],
+                paramLabel = "<j.fasta>",
+                required = true
+            )
+            override var pathOps: Path? = null
+                set(value) {
+                    ValidationException.requireFileType(value, InputFileType.FASTA)
+                    field = value
+                }
+
+            @set:Option(
+                names = ["--c-gene-feature"],
+                description = ["Gene feature corresponding to Constant gene sequences in FASTA (CExon1 by default)"],
+                paramLabel = "<gene_feature>",
+                required = false
+            )
+            override var geneFeatureOps: String? = "CExon1"
+                set(value) {
+                    ValidationException.requireKnownGeneFeature(value)
+                    field = value
+                }
+        }
+
+        @set:Option(
+            names = ["--c-genes-from-species"],
+            description = ["Species to take Constant genes from it (human, mmu, lamaGlama, alpaca, rat, spalax"],
+            paramLabel = "<species>",
+            required = true
+        )
+        override var species: String? = null
+            set(value) {
+                ValidationException.requireKnownSpecies(value)
+                field = value
+            }
+    }
+
+    @set:Option(
+        names = ["--species"],
+        description = ["Species names"],
+        paramLabel = "<species name>",
+        required = true,
+
+        )
+    var species: String = ""
+
+    @set:Option(
+        names = ["--taxon-id"],
+        description = ["Taxon ID"],
+        paramLabel = "<taxon_id>",
+    )
+    var taxonId: String = "0"
+
+    @set:Option(
+        names = ["--chain"],
+        description = ["Immunological chain"],
+        paramLabel = "<chain>",
+        required = true
+    )
+    var chain: String = ""
+        set(value) {
+            ValidationException.requireKnownChains(value)
+            field = value
+        }
+
+    @set:Option(
+        names = ["--do-not-infer-points"],
+        description = ["Do not infer reference points"],
+        required = false
+    )
+    var doNotInferPoints: Boolean = false
```

Comment:
Maybe invert logic? And add `showDefaultValue = ALWAYS`

Reply:
I mean `var inferPoints: Boolean = true`

Reply:
@gnefedev 
Still do not understand. In that case how to specify "NOT"? I tries doing:
```kotlin
@Option(names = ["--infer-points"], showDefaultValue = ALWAYS, required = false)
var inferPoints: Boolean = true
```

And in help I have:
```
--infer-point     Do not infer reference points
                  Default: null
```
How then user can specify to NOT to infer points?

Reply:
yes, sorry, it will be incovinient: `--infer-points false`, it may work, but doesn't have seance